returns/interfaces/specific/future.py

```diff
@@ -6,6 +6,8 @@
 :class:`returns.interfaces.specific.future_result.FutureResultBasedN` type.
 """
 
+from __future__ import annotations
```

Comment:
I see that you don't change `'Future[_UpdatedType]'` to `Future[_UpdatedType]`
Maybe, that's the problem?

Reply:
I'm happy to say, you're right 🥇

Reply:
I'll adjust this for all files containing `annotations` import

Reply:
Ok, I can't do this for all files!!
I got some errors about forward references:
```text
WARNING: Cannot resolve forward reference in type annotations of ...
```

Reply:
I think I have found the "perfect" balance

Reply:
I found what I did wrong, my bad. I was removing the apostrophes from types restricted to type checking

Reply:
Btw, I can't remove the apostrophes from `'Future[_UpdatedType]'` because `Future` is restricted to type checking in this file